src/AutoSaveStatus/AutoSaveStatusProvider.tsx

```diff
@@ -1,26 +1,27 @@
 import React, { PropsWithChildren, useCallback, useEffect, useMemo, useRef, useState } from "react";
 
 export enum AutoSaveStatus {
+  /** No calls are in-flight or just-recently-saved. */
   IDLE = "idle",
+  /** A call is actively in-flight. */
   SAVING = "saving",
+  /** A call is no longer actively in-flight, but has recently finished/can show confirmed. */
   DONE = "done",
+  /** A call is no longer actively in-fight, but has errored out. */
   ERROR = "error",
 }
 
 export interface AutoSaveStatusContextType {
   status: AutoSaveStatus;
-  /** Resets status to IDLE, particularly useful if "Error" or "Done" is stale */
-  resetStatus: VoidFunction;
-  errors: unknown[];
+  errors: string[];
```

Comment:
I changed the type to `string[]` b/c one place was passing Errors, another strings, and to be useful to display, the client needs to have some sort of idea of what's here.